examples/gallery/line/vector-heads-tails.py

```diff
@@ -0,0 +1,44 @@
+"""
+Vector heads and tails
+-------
+
+The :meth:`pygmt.Figure.plot` method can plot vectors with individual 
+heads and tails. We must specify the modifiers (together with the vector type)
+by passing the corresponding shortcuts to the  ``style`` argument. 
```

Comment:
Many GMT modules can plot vectors. The modifiers (e.g, +b, +e) shown in this gallery also work in other modules. Please try to rephrase the sentence to make it more general.

It would be better if you could explain what **+b** **+e** means, and **A**, **I** et al.
See https://docs.generic-mapping-tools.org/latest/plot.html#vector-attributes for reference.

Reply:
Made some modifications, thanks for the suggestions!